packages/pds/tests/seeds/client.ts

```diff
@@ -197,7 +206,71 @@ export class SeedClient {
     return repost
   }
 
-  userRef(did: string): UserRef {
+  async createScene(creator: string, handle: string) {
+    const res = await this.client.app.bsky.actor.createScene(
+      {
+        handle,
+      },
+      {
```

Comment:
would be nice to make the types a bit friendlier here, but that's for another day

Reply:
Just curious— where's the unfriendly type here?

Reply:
the same that you pointed it. it doesn't allow partial opts. so you can either provide _both_ headers & encoding or _neither_

Reply:
Ahh, got it!